src/elmProjectManager.ts

```diff
@@ -0,0 +1,9 @@
+import * as vscode from 'vscode';
+import { ElmProjectManager } from 'elm-project-inspect';
+import * as _ from 'lodash';
+
+export const getGlobalProjectManager = _.memoize(() => {
+  return new ElmProjectManager(
+    (vscode.workspace.workspaceFolders || []).map(x => x.uri.fsPath),
```

Comment:
Haven't experimented with this much. There may be an event to indicate when the workspace folders change.